packages/cashier/src/app.jsx

```diff
@@ -13,11 +12,9 @@ const App = ({ passthrough: { WS, root_store } }) => {
     }, []);
 
     return (
-        <MobxContentProvider store={root_store}>
-            <StoreProvider store={root_store}>
-                <Routes />
-            </StoreProvider>
-        </MobxContentProvider>
+        <StoreProvider store={root_store}>
```

Comment:
```suggestion
        <StoreProvider>
```
Later when we could move all stores to the `@deriv/stores` we won’t need to pass anything to the provider anymore as the stores already live in the `@deriv/stores` package.